felt/Cargo.toml

```
@@ -12,6 +12,7 @@ default = ["std"]
 std = []
 alloc = []
 lambdaworks-felt = ["dep:lambdaworks-math"]
+fuzzing = ["arbitrary", "num-bigint/arbitrary"]
```

I think we should call this feature "arbitrary" so it's easier to understand what it does.
Also, we should keep in mind the use of non-`dep:` optional features. We recently had a PR that added it to the optional features, and it cut the CI time from ~30 mins to ~8.
```suggestion
arbitrary = ["dep:arbitrary", "num-bigint/arbitrary"]
```

---

Resolved: https://github.com/lambdaclass/cairo-vm/commit/0f8c4b14a0c4532203bb6a27da79209af1c9a729